src/v2/Apps/Order/Components/ArtworkSummaryItem.tsx

```diff
@@ -31,9 +31,14 @@ const ArtworkSummaryItem: React.FC<ArtworkSummaryItemProps> = ({
   },
   ...others
 }) => {
-  const artwork = get({}, () => lineItems?.edges?.[0]?.node?.artwork)
+  const firstLineItem = get({}, () => lineItems?.edges?.[0]?.node!)
+  const { artwork, artworkVersion } = firstLineItem!
 
-  const { artistNames, title, date, shippingOrigin, image } = artwork!
+  if (!artwork) console.warn("artwork missing")
+  if (!artworkVersion) console.warn("artwork version missing")
+
+  const { artistNames, title, image } = artworkVersion || {}
+  const { date, shippingOrigin } = artwork || {}
```

Comment:
I'm a bit confused by this, could you clarify the difference between `artwork` and `artworkVersion`?

Reply:
When an artwork is updated we sometimes [create a new `ArtworkVersion` record](https://github.com/artsy/gravity/blob/main/app/models/domain/artwork.rb#L1542), which is like a snapshot of some core fields of the artwork at that time. Both are attached to the exchange order, so we can access those fields on the artworkVersion which remains visible after the artwork is unpublished.

The [actual logic where we do this](https://github.com/artsy/gravity/blob/main/app/models/domain/artwork.rb#L1361-L1365) only gets [called sometimes](https://github.com/artsy/gravity/search?q=.versioned) which makes things a little more confusing - it's not just a rails `after_save` hook or something like that.

Reply:
Thank you this makes a lot more sense now!